artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java

```diff
@@ -156,6 +160,10 @@ private void printStats(String result) {
       for (int i = 0; i < array.size(); i++) {
          printQueueStats(array.getJsonObject(i));
       }
+
+      if (maxRows == null && array.size() == DEFAULT_MAX_ROWS) {
```

Comment:
first of all array.size() == DEFAULT_MAX_ROWS seems duff logic, surely if size > DEFAULT_MAX_ROWS is the logix expected based on the output below.

surely better to have this as if maxRows >= array.size() this way even if someone sets it to some other value, thats still smaller than array.size then you want to ouytput that the output has been limited.

Reply:
The maxRows should always be >=  array.size() but I would like to print the warning only if maxRows is not set by user and array.size() == DEFAULT_MAX_ROWS.
If you call `queue stat` without the parameter `maxRows` then you get at most 50 queues but you could not know that results are limited by the default maxRows value.
The purpose of the warning is only to help users that don't know that default maxRows value is 50.